### DISCUSSION TITLE
Any way to see exact timestamp of music?

### DISCUSSION THREAD

**Discussion by NikunjK8** [namidaco/namida#182] [Q&A]
Like is there any way to see exact minutes/seconds of where the song is during seeking instead of this indicator which shows added up or subtracted value?
![IMG_20240502_103902](https://github.com/namidaco/namida/assets/99175083/45039477-cb4b-4ac7-b2ad-1994c0177e04)

**Comment by NikunjK8** [comment]
On a side note, the app adapts color from album cover art when a different song is playing. Is this by design?

![Screenshot_2024-05-02-10-37-36-518_com msob7y namida](https://github.com/namidaco/namida/assets/99175083/482f64ab-444a-4b8a-900c-b128323c5c74)

**Comment by MSOB7YY** [comment]
as for the exact seconds, no. would be an overkill to have an option for that

as for adapting color from album page, yeah it's intentional, is it bad?

**Comment by NikunjK8** [comment]
Yeah, just wanted to ask

Nope. It's only very noticeable when both colors are quite opposite as shown in the screenshot. Otherwise it's good.

**Reply by MSOB7YY** [reply]
i see, sorry yeah color can bother sometimes but having an option for that is kinda uuhh

**Reply by NikunjK8** [reply]
No extra option needed. It's rare as i mentioned and i think it's a thoughtful design.
